spec/models/manageiq/providers/microsoft/infra_manager/provision_workflow_spec.rb

```diff
@@ -16,6 +16,27 @@
     described_class.new({}, admin.userid)
   end
 
+  describe "#load_hosts_vlans" do
+    let(:hosts)       { [FactoryGirl.create(:host_with_ref)] }
```

Comment:
You only uses hosts as an array once, but do `hosts.first` like 4 times, maybe just wrap it in `[host]` once?